kobolds/sociology.md

```diff
@@ -99,6 +99,10 @@ the mouth is a potential target.
 
 ### Family
 
+Eggs and young are cared for communally generally with a group of trained,
+dedicated kobolds leading the task, rather than the birth parents. See below for
+more of Kobold family formation.
```

Comment:
This is just dumped here. It'll be more elegantly folded int when this doc gets rearranged.